x-pack/license-tools/src/test/java/org/elasticsearch/license/licensor/TestUtils.java

```diff
@@ -35,7 +37,7 @@
     public static final String PUBLIC_KEY_RESOURCE = "/public.key";
     public static final String PRIVATE_KEY_RESOURCE = "/private.key";
 
-    private static final DateFormatter dateFormatter = DateFormatters.forPattern("yyyy-MM-dd");
+    private static final DateFormatter dateFormatter = DateFormatters.forPattern("yyyy-MM-dd").withZone(ZoneId.systemDefault());
```

Comment:
let's use `ZoneOffset.UTC` here to have stable tests among all developers across the world